rpc/lib/server/http_server.go

```diff
@@ -27,16 +27,25 @@ const (
 	// maxBodyBytes controls the maximum number of bytes the
 	// server will read parsing the request body.
 	maxBodyBytes = int64(1000000) // 1MB
+
+	// same as the net/http default
+	maxHeaderBytes = 1 << 20
```

Comment:
Go will use a default in the absence of such. Is there a reason to duplicate on our side?

Reply:
Just to be explicit.